### PR TITLE
Reduce renders in User Name when adding a user

### PR DESCRIPTION
**Pull request by cniackz**
## What does this do?

Reduce renders in User Name when adding a user (Notice only that field is being rendered, not the entire screen):

reduce-renders

### CODE REVIEW

File: portal-ui/src/screens/Console/Users/AddUserScreen.tsx [hunk 1]
```diff
@@ -119,7 +126,7 @@ const AddUser = ({ classes }: IAddUserProps) => {
 
   const resetForm = () => {
     setSelectedGroups([]);
-    setAccessKey("");
+    setUserName("");
```
Comment (dvaldivia) [inline portal-ui/src/screens/Console/Users/AddUserScreen.tsx:129]: you probably need to `dispatch()` this, and in that case you can start wariting a `resetForm()` action on the slice that resets the whole state to `initialState`

**Review by dvaldivia — DISMISSED** [review]

**Comment by cniackz** [comment]
Wait, this is breaking the `Clear` functionality and I still don't know why, will continue tomorrow, I am tired of trying with no success for many hours... Will ping Daniel tomorrow to get some advice on this. Mui Button onclick property.

**Comment by cniackz** [comment]
Cannot clear: 

Screen Shot 2022-06-08 at 9 01 45 PM

**Comment by cniackz** [comment]
By doing this activity and spending the whole day, I have learned more of Redux/React than any other activity I have done so far. Now it is clear to me how to states are managed and how things are organized. I am becoming better at this. By the way, it is not the same reading or taking small tuto than performing this type of activities, amazing! And glad to contribute with this project. 👍

**Comment by cniackz** [comment]
By the way:

1. Creating a user -> PASS
2. Clearing the fields (resetForm) -> PASS
3. Selecting Policies -> PASS

I think this change isn't breaking current functionality.
Please double check!

**Comment by cniackz** [comment]
Please don't review this for now, I am about to add more changed to complete this PR.

**Review by dvaldivia — APPROVED** [review]

**Comment by cniackz** [comment]
I will resolve the conflict later today or tomorrow morning.

**Comment by cniackz** [comment]
Please don't approve, there is a bug in this PR.
After adding the user, the page stay on the same place but it should move back to user.
Need to investigate why...

**Review by dvaldivia — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Pull request merged**